smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/StructureGenerator.java

```diff
@@ -53,21 +55,26 @@ public void run() {
     /**
      * Renders a normal, non-error structure.
      *
-     * <p>For example, given the following Smithy model:
+     * <p>
```

Comment:
There're a ton of formatting changes in this file that are at either PR noise or something we don't want changed. Can they be reverted?

Reply:
Was added by mistake as smithy-typescript doesn't have auto formatting available on save like aws-sdk-js-v3 does. Will remove in next commit

Reply:
Formatting removed in [commit](https://github.com/awslabs/smithy-typescript/pull/183/commits/c948214e41ce11721931182283ad491c8cb5adb0)